bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/ModelParser.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
because the PR about the headers isn't merged. I can update whatever PR is going to be merged last.